terraform/utils.go

```diff
@@ -60,13 +67,9 @@ func tfConfigList(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateDat
 
 	// Fail if no paths are specified
 	terraformConfig := GetConfig(d.Connection)
-
-	// Should we still consider this as a required argument ??
-	// Since the plugin now supports parsing TF plan as well. What will happen if the config has only plan_file_paths defined? Should we return error ??
-
-	// if terraformConfig.ConfigurationFilePaths == nil {
-	// 	return nil, errors.New("configuration_file_path must be configured")
-	// }
+	if terraformConfig.Paths == nil && terraformConfig.ConfigurationFilePaths == nil && terraformConfig.PlanFilePaths == nil && terraformConfig.StateFilePaths == nil {
```

Comment:
We should instead return 0 rows for all tables if a user doesn't give any file paths (or none of the file paths match existing files)

Reply:
updated